chemprop/cli/train.py

```diff
@@ -910,6 +893,44 @@ def train_model(
                 args.weight_column,
                 args.no_header_row,
             )
+            input_cols = (args.smiles_columns or []) + (args.reaction_columns or [])
+            target_cols = columns[1:] if len(input_cols) == 0 else columns[len(input_cols) :]
+
+            individual_scores = dict()
+            for metric in model.metrics[:-1]:
```

Comment:
Why do you do `model.metrics[:-1]` instead of `model.metrics`?

Reply:
The last item in metrics is not a metric but a criterion, or the loss function

Reply:
The last one is the criterion for model training. I think we don't want to report this as a metric on the test set?